balboa-client-dispatcher/README.md

```diff
@@ -27,7 +27,7 @@ SBT:
  ```
  libraryDependencies ++= Seq(
    ...
-   "com.socrata" % "balboa-dispatcher-client" %% "[0.15, )",
+   "com.socrata" % "balboa-dispatcher-client" %% "[0.16, )",
```

Comment:
Will conflict with #23 but that's okay. Why different version specs in the two READMEs?